bots/tests-invoke

```diff
@@ -52,10 +52,14 @@ def main():
     name = os.environ.get("TEST_NAME", "tests")
     revision = os.environ.get("TEST_REVISION")
     test_project = os.environ.get("TEST_PROJECT")
+    # Set up to branch when overwritten with test name
```

Comment:
That sounds a bit hard to understand. How about "branch name when explicitly given in the test status"?